service/src/worker.rs

```diff
@@ -85,8 +97,17 @@ where
 		Ok(())
 	}
 
+	/// Returns a list of peers. The address of the worker self is replaced by ws://localhost.
```

Comment:
Could you add a note why this is necessary? Because for non-docker usage this doesn't make sense.

Reply:
That would be of interest for me as well, because it explains why we have this second change in the PR 😄 (which seems to be not obvious for a number of people)